face_width  : 960            # pixel dimensions
face_height : 720
face_xshift : 0              # centered in screen
face_model  : "GiGo"         # no eyelashes

face_skin   : 0x60A060       # light green head
face_back   : 0x000000       # black backdrop
face_brows  : 0xA04040       # brown eyebrows
face_eyes   : 0x000000       # black eye surrounds
face_iris   : 0x00C000       # dark green eye center 
face_stare  : 0x00E0FF       # light blue for stare
face_mouth  : 0xFF00FF       # magenta mouth 

voice_freq  : 105            # male fundamental
voice_infl  : 15             # some inflection
voice_shift : 100            # adult formants
voice_slow  : 120            # medium slow
voice_drama : 100            # normal excitability
voice_loud  : 100            # max volume


